compiler/optimizer/VPHandlers.cpp

```diff
@@ -10748,7 +10748,7 @@ static TR::Node *constrainCmplessthan(OMR::ValuePropagation *vp, TR::Node *node,
           performTransformation(vp->comp(), "%sChanging node [%p] %s into constant %d\n", OPT_DETAILS, node, node->getOpCode().getName(), result))
          {
          vp->removeChildren(node);
-         TR::ILOpCodes op = /*isUnsigned ? TR::iuconst : */TR::iconst;
+         TR::ILOpCodes op = TR::iconst;
```

Comment:
can you just put iconst directly into the recreate to save a needless local?

Reply:
Fixed in e4cb262